build/export-exchanges.js

```diff
@@ -517,11 +517,48 @@ function getErrorHierarchy() {
 
 // ----------------------------------------------------------------------------
 
+function generateErrorsTs () {
```

Comment:
@ttodua I'm wondering if this should be done here: https://github.com/ccxt/ccxt/blob/master/build/transpile.js#L1909

where we do the same for python and php

Reply:
iirc in export-exchanges it was already needed to exist, so atm it can be left there for now. but if you think it makes diference, we can think. just i avoided possible further side-effects and decided to create (errorsHierarchy.ts) right there as it was initially (firstly) accessed.

Reply:
ok I think we cna keep it here for now